### PR TITLE
fixed IPv6 iptables rules for enabled firewalld

### PR DESCRIPTION
**Pull request by bboehmke**
This should address moby/moby#41861

### CODE REVIEW

File: iptables/iptables.go [hunk 1]
```diff
@@ -512,8 +512,14 @@ func filterOutput(start time.Time, output []byte, args ...string) []byte {
...
 	if firewalldRunning {
+		// select correct IP version for firewalld
+		ipv := Iptables
+		if iptable.Version == IPv6 {
+			ipv = IP6Tables
+		}
+
 		startTime := time.Now()
-		output, err := Passthrough(Iptables, args...)
+		output, err := Passthrough(ipv, args...)
```
Comment (arkodg) [inline iptables/iptables.go:522]: why not pass `iptable.Version` directly ?

Comment (bboehmke) [reply iptables/iptables.go:522]: because 
`iptable.Version = "IPV6"`
and 
`IP6Tables = "ipv6"`

Not sure if Firewalld likes uppercase.

**Review by thaJeztah — APPROVED** [review]
SGTM

**Comment by thaJeztah** [comment]
@arkodg PTAL

**Comment by fnkr** [comment]
Hi, this fixes #41861, can it be merged please?

**Comment by thaJeztah** [comment]
@cpuguy83 @arkodg @tiborvass ptal

**Pull request merged**